man/os-release.xml

```diff
@@ -317,6 +317,23 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>FALLBACK_HOSTNAME=</varname></term>
```

Comment:
hmm, i think the field in os-release file should be called DEFAULT_HOSTNAME= actually, not FALLBACK_HOSTNAME. I think it should have a wider scope that the env var. i.e. this field should be shown in a graphical installer as the default hostname to set, and if it is set this should be propagated down.

hence, the env var that is specifiy to systemd, and controls very narrow system behaviour should be SYSTEMD_FALLBACK_HOSTNAME, but the much more generic field on os-release that might be useful for in installers, configuration tools, … should be DEFAULT_HOSTNAME.